tools/travis/k8s-tools.sh

```diff
@@ -17,19 +17,21 @@
 
 set -e
 
+GOPATH="$(git rev-parse --show-toplevel):${GOPATH}"
```

Comment:
I fixed the build problem with the last commit. can you please revert your changes for this file or at least the GOPATH

Reply:
done